internal/command/config/env.go

```diff
@@ -47,6 +48,16 @@ func runEnv(ctx context.Context) error {
 		return err
 	}
 
+	app, err := apiClient.GetAppCompact(ctx, appName)
+	if err != nil {
+		return fmt.Errorf("failed to get app: %s", err)
+	}
+
+	ctx, err = apps.BuildContext(ctx, app)
```

Comment:
I think using `flaps.New` and `flaps.NewContext` will do equally well here? It's just that `apps.BuildContext` always starts up an agent for Wireguard, but we default to `api.machines.dev` now. (`flaps.New` itself will start an agent when it needs it.)

Reply:
you're so right. thanks!